plugin.rb

```diff
@@ -75,7 +75,10 @@ class Engine < ::Rails::Engine
       }
     end
 
-    likes = object.post_actions.where('deleted_at IS NULL AND post_action_type_id = ?', PostActionType.types[:like])
```

Comment:
Records have already been eager loaded so we should not calling ActiveRecord query methods on the collection proxy.